test/plugin_functional/test_suites/core_plugins/application_status.ts

```diff
@@ -26,6 +27,15 @@ import {
 import { PluginFunctionalProviderContext } from '../../services';
 import '../../plugins/core_app_status/public/types';
 
+const getKibanaUrl = (pathname?: string, search?: string) =>
+  url.format({
+    protocol: 'http:',
+    hostname: process.env.TEST_KIBANA_HOST || 'localhost',
+    port: process.env.TEST_KIBANA_PORT || '5620',
+    pathname,
+    search,
+  });
```

Comment:
So, this is the third duplication of this helper in `test/plugin_functional/test_suites/core_plugins/*` files. I would gladly extract it to an util file, however I'm unsure of the conventions for such files in FTR code. Moving it to a service seems overkill as it's 'stateless'. Any idea?

Reply:
FTR already uses utils decalred in https://github.com/elastic/kibana/blob/8e9a8a84dccfa7965ce8a22362885e6cdef8b51f/src/test_utils/get_url.js

you can extend it